pythonforandroid/recommendations.py

```diff
@@ -187,7 +187,7 @@ def check_ndk_api(ndk_api, android_api):
 
 
 MIN_PYTHON_MAJOR_VERSION = 3
-MIN_PYTHON_MINOR_VERSION = 4
+MIN_PYTHON_MINOR_VERSION = 6
```

Comment:
Aaaa...you made this change at #1975, see [AndreMiras comment](https://github.com/kivy/python-for-android/pull/1975/files#r319776002)...anyway, leave it if you want, just remember to remove it from there :wink:

Reply:
It doesn't matter where this change comes from, it just needs to be made.